torch/_higher_order_ops/flex_attention.py

```diff
@@ -756,11 +761,42 @@ def flex_attention_autograd(
     return out, logsumexp
 
 
+@flex_attention.py_impl(_FlopCounterMode)
+def flex_attention_flop_counter_mode(
+    mode: _FlopCounterMode,
+    query: torch.Tensor,
+    key: torch.Tensor,
+    value: torch.Tensor,
+    score_mod: Callable,
+    block_mask: tuple,
+    scale: float,
+    kernel_options: dict[str, Any],
+    score_mod_other_buffers: tuple = (),
+    mask_mod_other_buffers: tuple = (),
+) -> tuple[torch.Tensor, torch.Tensor]:
+    for par in set(mode.counter.mod_tracker.parents):
+        mode.counter.flop_counts[par]["flex_attention"] += sdpa_flop_count(
+            query.shape, key.shape, value.shape
+        )
+    out, lse = math_attention(
```

Comment:
I don't think we want this. It should actually run flex attention here, at least as long as it's a compatible device (CUDA).